Lib/test/test_random.py

```diff
@@ -365,6 +365,8 @@ def test_getrandbits(self):
         self.assertRaises(TypeError, self.gen.getrandbits, 1, 2)
         self.assertRaises(ValueError, self.gen.getrandbits, -1)
         self.assertRaises(TypeError, self.gen.getrandbits, 10.1)
+        self.assertRaises(TypeError, self.gen.getrandbits, 'test')
```

Comment:
These tests already pass for me, without your changes - you should assert on the specific message